### PR TITLE
Run code through clang-format 9.0.

### PR DESCRIPTION
**Pull request by henrycg**
This is a revival of @degregat 's PR #88 after merging the recent PRs.

Generated by running the following command in the top-level dir:

> clang-format --style=Mozilla \
   -i include/*.h pclient/*.c prio/*.c prio/*.h ptest/*.c ptest/*.h

### CODE REVIEW

**Comment by degregat** [comment]
https://github.com/degregat/libprio/blob/8329ba870ba38b2c7b04f06dedd41958e022ae61/scripts/clang-format.sh

and 

https://github.com/degregat/libprio/blob/8329ba870ba38b2c7b04f06dedd41958e022ae61/.travis.yml

might help.

File: .travis.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ addons:
     - scons
     - libmsgpack-dev
     - libnss3-dev
-    - clang-format-3.9
+    - clang-9
```
Comment (degregat) [inline .travis.yml:27]: clang-format-9 should be correct here

File: scripts/clang-format.sh [hunk 2]
```diff
@@ -8,20 +8,23 @@ die() {
     exit 1
 }
 
-CLANG_FORMAT_VERSION="clang-format version 3.9"
+CLANG_FORMAT_VERSION="clang-format version 9.0"
 
-if which clang-format-3.9 > /dev/null; then
-    alias clang-format=clang-format-3.9
+if which clang-format-9.0 > /dev/null; then
```
Comment (degregat) [inline scripts/clang-format.sh:13]: here it should be clang-format-9 too

**Comment by rhelmer** [comment]
@henrycg would you mind sharing the script you used to generate this? I see a few differences when I run it locally but it might be because homebrew just updated to `clang-format` v10 :/

It looks pretty minor, and I don't think we should spin our wheels too much expecting consistent tooling across platforms, but would like to make sure I didn't miss a step. I'll grab an ubuntu VM w/ `clang-format-9` and see what that produces too. Thanks!

**Comment by henrycg** [comment]
@rhelmer I ran this on MacOS with macports using
> `clang-format-mp-9.0 --style=Mozilla -i include/.h pclient/.c prio/.c prio/.h ptest/.c ptest/.h`

**Comment by rhelmer** [comment]
> @rhelmer I ran this on MacOS with macports using
> 
> > `clang-format-mp-9.0 --style=Mozilla -i include/.h pclient/.c prio/.c prio/.h ptest/.c ptest/.h`

Thanks. Huh, this is a silly PR to get hung up on :) But I'm just not seeing the same formatting applied from `clang-format-9` on Ubuntu Bionic or the latest `clang-format` (10) from homebrew on macOS. I'll take a look tomorrow, I wonder if the macport version is the odd one out.

**Comment by rhelmer** [comment]
OK sorry for the delay, I figured things out on my end. I think there might be some annoying differences between `clang-format` 9 and 10 but let's cross that bridge when we come to it.

**Pull request merged**